tests/CMakeLists.txt

```diff
@@ -32,25 +32,20 @@ if(ENABLE_SSH OR ENABLE_TLS)
     endif()
 endif()
 
-foreach(src IN LISTS libsrc)
-    list(APPEND test_srcs "../${src}")
-endforeach()
-add_library(testobj OBJECT ${test_srcs})
```

Comment:
FIXME: this does not work with older cmake versions!

Reply:
You mean `add_library(<name> OBJECT <sources>)`? I found it in the documentation of cmake 2.8.12 so it should work.